analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -73,9 +79,54 @@ impl<'a> Reader<'a> {
             batch_size,
             meta_data: None,
             row_projector: None,
+            parallelism_options,
         }
     }
 
+    async fn read_parallelly(
```

Comment:
How about name this:
```
try_parallel_read
```

Reply:
Why modify `read_parallelly` to `try_parallel_read`? The return value is `Result` cannot decide to use the `try` prefix.

Reply:
How about `suggest_to_read_parallelly`? 
Try prefix seems to emphasize the `Result` returned value?